frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/vm-settings-tab.tsx

```diff
@@ -53,6 +62,52 @@ export class VMSettingsTabComponent extends React.Component<VMSettingsTabCompone
       goToStep,
     } = this.props;
 
+    const provisionSourceValue = this.getFieldValue(VMSettingsField.PROVISION_SOURCE_TYPE);
```

Comment:
there are many fields in General Tab and we try to keep this component short as possible. Please either make a new component for this message or make a new component for provision source which will include this functionality.

Reply:
makes sense to me too :+1:

I can make a follow up PR to move the "source" input into a separate component.

Reply:
+1